inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -51,7 +61,8 @@ def simulate(
             simulation_time_ns: float = 10,  # ns
             output_timestep_ps: float = 1,  # ps
             integrator: Literal["md", "sd", "bd"] = "md",
-            n_steps_min: int = 5000) -> tasks.Task:
+            n_steps_min: int = 5000,
+            compressed_trajectory: bool = False) -> tasks.Task:
```

Comment:
I don't think this decision should be made here but I would like to leave a comment on the naming.
The name of this parameter, as it is, is non-informative and ambiguous. For this kined of parameters, try to follow the pattern

VERB_OBJECT

for example something like:

generate_compressed_trajectory

would be much easier to understand.